HISTORY.rst

```diff
@@ -3,6 +3,24 @@ Release notes
 =============
 
 
+2.9.1 (2021-03-22)
+------------------
+
+* This is relatively large change, it could have justified a major version bump,
+  but the previous addition was relatively minor. So going to hold off on bumping major yet.
```

Comment:
> but the previous addition was relatively minor

I do not understand that - what "previous addition" is meant here?

Reply:
Version 2.9.0 was retiring the optional `classifiers.txt` file used to auto-fill `classifiers=` -> that is relatively minor I think (because not used, and provides a minimal added value)

Reply:
I was going to bump to 3.0.0 BTW, but tentatively holding off :) WDYT? Should I bump to 3.0.0?
(I was hoping to put in more versioning strategies in 3.0, like a "date based" one that would yield `CCYY-MM-DD`, similarly to https://pypi.org/project/regex/ for example)

Reply:
Rephrased it now, I think you're right: the initial phrasing was quite poor :)